### PR TITLE
Add pagination to bdb detail events

### PR DESCRIPTION
**Pull request by LudvigHz**
bdb detail Currenty Gives 404 error when fetching events for company. This adds pagination to fix this.

### CODE REVIEW

File: app/routes/bdb/BdbDetailRoute.js [hunk 1]
```diff
@@ -35,23 +48,38 @@ const mapStateToProps = (state, props) => {
...
+  const { companyId } = props.params;
+  const fetchMoreEvents = () =>
+    dispatch(
+      fetchEventsForCompany({
+        queryString: queryString(companyId),
+        loadNextPage: true
+      })
+    );
+  return {
+    deleteSemesterStatus,
```
Comment (odinuge) [inline app/routes/bdb/BdbDetailRoute.js]: Have you tested that this works?

Comment (LudvigHz) [reply app/routes/bdb/BdbDetailRoute.js]: nope!

Comment (odinuge) [reply app/routes/bdb/BdbDetailRoute.js]: :100:
Aka. PR is WIP?

Comment (LudvigHz) [reply app/routes/bdb/BdbDetailRoute.js]: Should be fixed now!

File: app/routes/bdb/BdbDetailRoute.js [hunk 2]
```diff
@@ -35,23 +48,40 @@ const mapStateToProps = (state, props) => {
...
+  const { companyId } = props.params;
+  const fetchMoreEvents = () =>
+    dispatch(
+      fetchEventsForCompany({
+        queryString: queryString(companyId),
+        loadNextPage: true
+      })
+    );
+  return {
+    deleteSemesterStatus: (companyId, semesterStatusId) =>
```
Comment (odinuge) [inline app/routes/bdb/BdbDetailRoute.js]: 10/10 code duplication. Looks kinda wanky too. :man_shrugging:  You can use `foo: (...args) => dispatch(foo(...args)`, or even better use `bindActionCreators`.

**Pull request merged**